java/runtime/src/main/java/org/ray/runtime/functionmanager/FunctionManager.java

```diff
@@ -191,6 +191,11 @@ RayFunction getFunction(JavaFunctionDescriptor descriptor) {
           RayFunction rayFunction = new RayFunction(e, classLoader,
               new JavaFunctionDescriptor(className, methodName, typeDescriptor));
           map.put(ImmutablePair.of(methodName, typeDescriptor), rayFunction);
+          // For cross language call java function without signature
+          final Pair<String, String> emptyDescriptor = ImmutablePair.of(methodName, "");
```

Comment:
Line195 should be at the `if` block since it is only used for `if` block.

Reply:
empyDescriptor will be used in `if` condition, so it can't be moved to `if` block.